platform/api/openstack/api.go

```diff
@@ -628,5 +646,17 @@ func (a *API) GC(gracePeriod time.Duration) error {
 			return fmt.Errorf("couldn't delete server %s: %v", server.ID, err)
 		}
 	}
+
+	images, err := a.listImagesWithTags([]string{"mantle"})
+	if err != nil {
+		return fmt.Errorf("listing Mantle images: %w", err)
+	}
+
+	for _, image := range images {
+		if err := a.DeleteImage(image.ID); err != nil {
+			return fmt.Errorf("deleting image with name: %s", image.Name)
```

Comment:
I think the gc functions should continue to delete as much as possible but since the server deletion errors out early the same way it's ok to have it consistent.

Reply:
Yeah, I got the same thought - and garbage collector job is running periodically so it's ok if it fails once.

Reply:
It looks like it's trying to remove all the images with the "mantle" tag. Should there be a check if image is used or, I dunno, remove only images older than some days?

Reply:
Oh, right, can we do something like done for servers and check `.Created.After(threshold)`?

Reply:
Good idea - it seems there is no way to detect if an image is being used. Let's add the threshold!